extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -267,8 +267,20 @@ export class AmpScrollableCarousel extends BaseCarousel {
   cancelTouchEvents_() {
     // TODO(aghassemi, #4754): Ideally we only stop propagation of horizontal
     // touchmove events.
-    this.element.addEventListener('touchmove', event => {
-      event.stopPropagation();
-    });
+
+    // We don't preventDefault, tell the browser to optimize scrolling.
+    const options = {
+      passive: true,
+    };
+
+    /**
+     * TODO(aghassemi, #5105): Type check
+     * @suppress {checkTypes}
+     */
+    (() => {
```

Comment:
this way I can get @suppress to only apply to that code (I tried to do this with code blocks in {} too but looks like these annotation only work at function scope)